vcd/datasource_vcd_lb_service_monitor.go

```diff
@@ -0,0 +1,90 @@
+package vcd
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func datasourceVcdLbServiceMonitor() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceVcdLbServiceMonitorRead,
+		Schema: map[string]*schema.Schema{
+			"vdc": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"edge_gateway": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Edge gateway name",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Service Monitor name",
+			},
+			"interval": &schema.Schema{
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"timeout": &schema.Schema{
```

Comment:
I can add them but you will never see descriptions in datasource while using the GUI. Resource does have them.

Reply:
I asked because two properties have descriptions. 
Either we duplicate the descriptions for all, or we add a comment saying that descriptions are listed under the resource definition

Reply:
Ok. That's true for these two. Can we agree I add for the "non-computed" ones (then I'll add for `org` and `vdc`)? As these make sense, they're required and if missed the description may be printed.
On the other hand I can easily copy over all of them, it's just that I don't expect them to be seen.

Reply:
Agreed

Reply:
Here's my take at it.